core/src/main/resources/hudson/model/View/popup.jelly

```diff
@@ -0,0 +1,43 @@
+<div class="remodal" data-remodal-id="modal">
```

Comment:
You mean where `hudson.model.AllView` class is located? I tried to put popup.jelly to folders model, model/View, model/AllView but with the same error. Header part of the form is inserted without errors, but a bigger part crashes.